sandbox.rb

```diff
@@ -0,0 +1,45 @@
+#require "pry"
+
+class Arena# < ActiveRecord::Base
```

Comment:
Don't duplicate the classes here.  Just call them.  arena = Arena.new, etc.  Make sure you require the appropriate files, so that Arena is available here.